### PR TITLE
[UsePythonVersionV0] Migrated to Node10.

### PR DESCRIPTION
**Pull request by egor-bryzgalov**
**Task name**: UsePythonVersionV0

**Description**: Migrated UsePythonVersionV0 to Node 10.

Removed `strict` property from `tsconfig.json` for consistency. It allowed us to avoid unnecessary types casting.

**Documentation changes required:** No

**Added unit tests:** No

**Attached related issue:** [#360](https://github.com/microsoft/build-task-team/issues/360)

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/UsePythonVersionV0/Tests/L0.ts [hunk 1]
```diff
@@ -22,7 +22,9 @@ describe('UsePythonVersion L0 Suite', function () {
         require('./L0_versionspec');
     });
 
-    it('succeeds when version is found', function () {
+    it('succeeds when version is found', function (done: Mocha.Done) {
+        this.timeout(parseInt(process.env.TASK_TEST_TIMEOUT as string) || 5000);
```
Comment (anatolybolshakov) [inline Tasks/UsePythonVersionV0/Tests/L0.ts:26]: Do we need to insert timeout for every test? Could it be enough to mention it in describe?

Comment (egor-bryzgalov) [reply Tasks/UsePythonVersionV0/Tests/L0.ts:26]: Thanks! Let me try this.

Comment (egor-bryzgalov) [reply Tasks/UsePythonVersionV0/Tests/L0.ts]: Applied change for this.

File: Tasks/UsePythonVersionV0/Tests/L0.ts [hunk 2]
```diff
@@ -38,9 +40,10 @@ describe('UsePythonVersion L0 Suite', function () {
         assert(didPrependPath(testRunner, pythonBinDir));
         assert.strictEqual(testRunner.stderr.length, 0, 'should not have written to stderr');
         assert(testRunner.succeeded, 'task should have succeeded');
+        done();
```
Comment (anatolybolshakov) [inline Tasks/UsePythonVersionV0/Tests/L0.ts:43]: Do we need to use done here? There doesn't seem to be any asynchronous code

**Review by anatolybolshakov — APPROVED** [review]
LGTM, could you please take a look at comments also?

File: Tasks/UsePythonVersionV0/Tests/L0.ts (hunk 2 above)
Comment (egor-bryzgalov) [reply Tasks/UsePythonVersionV0/Tests/L0.ts]: Thanks! Resolved this.

**Review by max-zaytsev — APPROVED** [review]

**Comment by egor-bryzgalov** [comment]
Merge at 182 sprint

**Pull request merged**